src/Okta.Sdk.IntegrationTests/PoliciesScenarios.cs

```diff
@@ -0,0 +1,1482 @@
+                await client.Policies.DeletePolicyAsync(createdPolicy1.Id);
+                await client.Policies.DeactivatePolicyAsync(createdPolicy2.Id);
+                await client.Policies.DeletePolicyAsync(createdPolicy2.Id);
+            }
+        }
+
+        [Fact]
+        public async Task DeletePolicy()
+        {
+            var client = TestClient.Create();
+
+            var policy = new OktaSignOnPolicy()
+            {
+                // Name has a maximum of 50 chars
+                Name = $"Sign On policy {Guid.NewGuid()}".Substring(0, 50),
+                Type = PolicyType.OktaSignOn,
+                Status = "ACTIVE",
+                Description = "The default policy applies in all situations if no other policy applies.",
+            };
+
+            var createdPolicy = await client.Policies.CreatePolicyAsync(policy);
+            var retrievedPolicy = await client.Policies.GetPolicyAsync(createdPolicy.Id);
+            retrievedPolicy.Should().NotBeNull();
+
+            await client.Policies.DeactivatePolicyAsync(createdPolicy.Id);
+            await client.Policies.DeletePolicyAsync(createdPolicy.Id);
+
+            Func<Task> act = async () => await client.Policies.GetPolicyAsync(createdPolicy.Id);
+            act.Should().Throw<OktaApiException>();
+        }
+
+        [Fact]
+        public async Task UpdatePolicy()
+        {
+            var client = TestClient.Create();
+
+            var policy = new OktaSignOnPolicy()
+            {
+                // Name has a maximum of 50 chars
+                Name = $"Sign On policy {Guid.NewGuid()}".Substring(0, 50),
+                Type = PolicyType.OktaSignOn,
+                Status = "ACTIVE",
+                Description = "The default policy applies in all situations if no other policy applies.",
+            };
+
+            var createdPolicy = await client.Policies.CreatePolicyAsync(policy);
+
+            createdPolicy.Name = $"Updated Sign On policy {Guid.NewGuid()}".Substring(0, 50);
+            createdPolicy.Description = "This description was updated";
+            // NIT: it would be nicer to do this intead: client.Policies.UpdatePolicyAsync(createdPolicy);
+            await client.Policies.UpdatePolicyAsync(createdPolicy, createdPolicy.Id);
+
+            var updatedPolicy = await client.Policies.GetPolicyAsync(createdPolicy.Id);
+
+            try
+            {
+                updatedPolicy.Id.Should().Be(createdPolicy.Id);
+                updatedPolicy.Type.Should().Be(createdPolicy.Type);
+                updatedPolicy.Name.Should().StartWith("Updated");
+                updatedPolicy.Description.Should().Be("This description was updated");
+            }
+            finally
+            {
+                await client.Policies.DeactivatePolicyAsync(createdPolicy.Id);
+                await client.Policies.DeletePolicyAsync(createdPolicy.Id);
+            }
+        }
+
+        [Fact]
+        public async Task ActivatePolicy()
```

Comment:
Looks like we're testing both activate and deactivate in here, so I'd suggest updating the task title to reflect this.

Reply:
Yeah, I would like to create an `INACTIVE` policy from the beginning but that is not possible, I will rename the test.